src/pages/case-study/doctorlink-setting-up-a-design-system.js

```diff
@@ -0,0 +1,689 @@
+            <BlockRow
+              mobile={addPaddings('4', '4')}
+              smallTablet={addPaddings('4', '4')}
+              tablet={addPaddings('4', '4')}
+              desktop={addPaddings('6', '6')}
+              widthPerc="75%"
+            >
+              <FullWidthBlock data={normalise(data1)} />
+            </BlockRow>
+          )}
+        </Grid>
+      </GreyBackground>
+
+      <Grid>
+        {/* The prologue */}
+        {shouldRender(data2) && (
+          <BlockRow
+            alignCenter
+            mobile={addPaddings('5', '5')}
+            smallTablet={addPaddings('7', '7')}
+            tablet={addPaddings('7', '7')}
+            desktop={addPaddings('7', '8')}
+          >
+            <TextAndImageBlock data={normalise(data2)} />
+          </BlockRow>
+        )}
+
+        {/* Youtube Video */}
+        {shouldRender(data3) && <VideoBlock data={normalise(data3)} />}
+
+        {/* Finding common ground */}
+        {shouldRender(data4) && (
+          <BlockRow
+            mobile={addPaddings('4', '4')}
+            smallTablet={addPaddings('5', '5')}
+            tablet={addPaddings('6', '6')}
+            desktop={addPaddings('7', '7')}
+          >
+            <TextColumnsBlock data={normalise(data4)} />
+          </BlockRow>
+        )}
+      </Grid>
+
+      <BlueBackground>
+        <Grid>
+          {/* Seamless */}
+          {shouldRender(data5) && (
+            <BlockRow
+              mobile={addPaddings('4', '4')}
+              smallTablet={addPaddings('5', '5')}
+              tablet={addPaddings('6', '6')}
+              desktop={addPaddings('6', '6')}
+            >
+              <TextColumnsBlock data={normalise(data5)} colorReverse />
+            </BlockRow>
+          )}
+        </Grid>
+      </BlueBackground>
+
+      <Grid>
+        {/* A new visual language */}
+        {shouldRender(data6) && (
+          <BlockRow
+            mobile={addPaddings('4', '4')}
+            smallTablet={addPaddings('5', '5')}
+            tablet={addPaddings('6', '6')}
+            desktop={addPaddings('7', '6')}
+          >
+            <TextColumnsBlock data={normalise(data6)} />
+          </BlockRow>
+        )}
+
+        {/* Image - Continue / back */}
+        {shouldRender(data7) && (
+          <BlockRow>
+            <FullWidthBlock data={normalise(data7)} />
+          </BlockRow>
+        )}
+
+        {/* Hi I am Noto sans */}
+        {shouldRender(data8) && (
+          <BlockRow
+            rowReverse
+            alignCenter
+            mobile={addPaddings('5', '5')}
+            smallTablet={addPaddings('6', '6')}
+            tablet={addPaddings('7', '7')}
+            desktop={addPaddings('8', '8')}
+          >
+            <TextAndImageBlock data={normalise(data8)} />
+          </BlockRow>
+        )}
+
+        <ReversableDiv>
+          {/* Colour */}
+          {shouldRender(data9) && (
+            <BlockRow
+              mobile={{ bottom: '5' }}
```

Comment:
The value story at this key is going to be used in selecting an index from an array, I wouldn't use a string here.

Reply:
I don't understand what you mean here... 🤔

Reply:
works with strings!